src/components/Calendar/Calendar.tsx

```diff
@@ -49,7 +50,61 @@ const CalendarStyling = styled.div`
     }
 
     .fc .fc-toolbar-title {
-        font-size: 1.5rem;
+        font-size: min(5vw, 1.5rem);
+    }
+    .fc .fc-icon {
+        font-size: min(4vw, 1.2rem);
+    }
+
+    .fc .fc-col-header-cell-cushion,
+    .fc .fc-daygrid-day-number,
+    .fc .fc-timegrid-axis-cushion,
+    .fc .fc-timegrid-slot-label-cushion,
+    .fc .fc-daygrid-more-link,
+    .fc .fc-event-title,
+    .fc .fc-event-time {
+        font-size: min(4vw, 1rem);
+    }
+
+    // month view grid display
+    .fc .fc-daygrid-day-frame {
+        cursor: pointer;
+        min-height: min(20vw, 7rem);
+    }
+
+    // month view events
+    .fc-dayGridMonth-view .fc-daygrid-day-number {
+        font-size: min(3vw, 1rem);
```

Comment:
How come the proportions change here? so why is it 3:1 here and closer to 4:1 earlier?

Reply:
Similarly these are all empirical settings